Scala/Scala.sublime-syntax

```diff
@@ -140,12 +213,25 @@ contexts:
             - match: '\)'
               pop: true
             - include: pattern-match
-        - match: '=>|⇒|\b(if)\b'
+        - match: '\b(if)\b'
+          captures:
+            1: keyword.control.flow.scala
+          set:
+            - match: '=>|⇒'
```

Comment:
I thought about it.  In fact, I was going to make a change, but in practice highlighting the lambda `=>` differently from the version of the same operator is surprisingly helpful in reading code.  They *are* different semantics (the `case` version doesn't declare a function).  `keyword.operator` might make sense though.